### PR TITLE
8729 - changes necessary for renewal expiry date

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#8729

*Description of changes:*
- changes to show and set expiry date
- tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/store/state/state-model.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ export const stateModel: StateModelIF = {
       lastUpdateDateTime: null
     },
     expiryDate: '',
+    surrenderDate: '',
```
Comment (doug-lovett) [inline ppr-ui/src/store/state/state-model.ts]: surrenderDate is now defined in registration and registration.lengthTrust?

Comment (jordiwes) [reply ppr-ui/src/store/state/state-model.ts]: yes, I'm not sure why I did that. I have removed it

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Review by kialj876 — APPROVED** [review]
Looks good!

**Pull request merged**